trustme/__init__.py

```diff
@@ -128,6 +129,38 @@ def _identity_string_to_x509(identity):
     return x509.DNSName(alabel)
 
 
+EC_SUPPORTED = {}
+EC_SUPPORTED.update([(curve.name.upper(), curve) for curve in [
```

Comment:
Is there a way to enumerate all elliptic curve types automatically? Maybe `EllipticCurve.__subclasses__()`?